src/arvcamera.c

```diff
@@ -1130,6 +1130,11 @@ arv_camera_get_frame_rate_bounds (ArvCamera *camera, double *min, double *max, G
 					return;
 				}
 
+				if (max != NULL)
```

Comment:
If n_values <= 0, we still want to assign a value to min and max. What was the clang static analyzer warning ?

Reply:
The warning was about uninitiated values.
E.g. when [`arv_camera_get_frame_rate_bounds`](https://github.com/AravisProject/aravis/blob/master/src/arvcamera.c#L1110) gets called from [`arv_camera_set_frame_rate`](https://github.com/AravisProject/aravis/blob/master/src/arvcamera.c#L941), [these 'greater than' comparisons](https://github.com/AravisProject/aravis/blob/master/src/arvcamera.c#L1139) can be wrong.

`guint n_values` can only be >= 0. In that case:
- if `n_values` == 0: final values of `min` and `max` are 0 (maybe it would be better to set an error?)
- if `n_values` > 0: final values of `min` and `max` are > 0.

Reply:
I'm so stupid, I thought the patch were removing those lines, not adding them.